src/ossia-max/src/ossia-max.cpp

```diff
@@ -532,12 +481,32 @@ std::vector<std::string> parse_tags_symbol(t_symbol** tags_symbol, long size)
   return tags;
 }
 
-void ossia_max::discover_network_devices(ossia_max* x)
+void ossia_max::discover_network_devices(ossia_max*)
 {
   ossia_max::zeroconf_oscq_listener.browse();
   ossia_max::zeroconf_minuit_listener.browse();
   clock_delay(ossia_max::browse_clock, 100.);
 }
 
+void ossia_max::push_parameter_value(ossia::net::parameter_base* param, const ossia::value& val)
+{
+  if(!param_locks.contains(param))
```

Comment:
hm, we're sure that this can only be accessed from a single thread ? (thinking e.g. of overdrive mode where things seem to happen in a bit more... random ways)

Reply:
absolutely no.
At least it can be accessed by a network thread when receiving new value and by max main thread when user changes a value with a UI element.
In the former case, the received value might be forwarded to another object through the patcher and change its value. In that case I don't know which thread is alive. Maybe there is some magic around the `output_anything` Max's SDK method and the message is queued then output by max main thread (like the Qt's queued connexion for signal). 
Concerning overdrive, for me it doesn't have much to do with threading. Maybe I'm wrong but in my head overdrive is just stopping the message propagation inside patcher to let the audio engine compute sample to output. And this always leads to undefined behavior. So putting some effort in making it `defined` doesn't make much sense to me. But again maybe I'm wrong (remember I was assuming one week ago that Max isn't thread safe...)